### DISCUSSION TITLE
aviator

### DISCUSSION THREAD

**Discussion by anas0022** [community/community#134619] [Programming Help]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

<!DOCTYPE html>
<html>
<head>
    <title>Aviator Prediction Software</title>
</head>
<body>
    <h1>Aviator Prediction Software</h1>
    <label for="previousMultiplier">Enter Previous Multiplier: </label>
    <input type="number" step="0.01" id="previousMultiplier" name="previousMultiplier">
    <button onclick="predictNextMultiplier()">Predict Next Multiplier</button>
    <p id="result"></p>

    <script>
        function generateRandomMultiplier() {
            return (Math.random() * (10 - 1) + 1).toFixed(2);
        }

        function predictNextMultiplier() {
            const previousMultiplier = parseFloat(document.getElementById('previousMultiplier').value) || 1;
            const nextMultiplier = generateRandomMultiplier();
            document.getElementById('result').innerText = `Previous Multiplier: ${previousMultiplier}, Predicted Next Multiplier: ${nextMultiplier}`;
        }
    </script>
</body>
</html>
 here i have created a aviator prediction site but its crap can anyone help me with this

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @anas0022 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by Mahd25** [comment]
## Hello @anas0022 

I think you mean something like this!


``` html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Aviator Prediction Software</title>
</head>

<body>
    <input type="text" id="previousMultiplier" placeholder="Enter previous multiplier">
    <button onclick="predictNextMultiplier()">Predict Next Multiplier</button>
    <div id="result"></div>

    <script>
        function generateRandomMultiplier() {
            return (Math.random() * (10 - 1) + 1).toFixed(2);
        }

        function predictNextMultiplier() {
            const previousMultiplier = parseFloat(document.getElementById('previousMultiplier').value) || 1;
            const nextMultiplier = generateRandomMultiplier();
            document.getElementById('result').innerText = `Previous Multiplier: ${previousMultiplier}, Predicted Next Multiplier: ${nextMultiplier}`;
        }
    </script>
</body>
</html>
```

***I hope it was useful for you.***

**Reply by anas0022** [reply]
thanks bro,
 this is far better than what i created but it seems not good, because i want the prediction should be nearly same to the result on aviator,
i think is not possible but i am trying to make something like that
